parsers/JP.py

```diff
@@ -329,26 +340,19 @@ def fetch_price(
     return data
 
 
-def parse_dt(row):
+def parse_dt(row) -> datetime:
     """Parses timestamps from date and time."""
-    if "AM" in row["Time"] or "PM" in row["Time"]:
-        timestamp = (
-            arrow.get(
-                " ".join([row["Date"], row["Time"]]).replace("/", "-"),
-                "YYYY-M-D H:mm A",
-            )
-            .replace(tzinfo="Asia/Tokyo")
-            .datetime
+    date = row["Date"]
+    time = row["Time"]
```

Comment:
Would it not be easier to parse these directly using the date and time libraries and then just combine them instead of using string manipulation?

Reply:
It looks only `datetime` library has `strptime()` but there is no string parser in `date` and `time` standard libraries (ref. https://docs.python.org/3.10/library/datetime.html#strftime-strptime-behavior). I wonder if there's another method other than `datetime.strptime()`.

Reply:
`datetime.time` has it's own parsing functions, same goes for date.

https://docs.python.org/3.10/library/time.html#time.strftime

Reply:
`datetime.time` has their own parsing functions, same with date.

https://docs.python.org/3.10/library/time.html#time.strptime